docs/sql-ref-syntax-qry-select.md

```diff
@@ -159,3 +159,6 @@ SELECT [ hints , ... ] [ ALL | DISTINCT ] { named_expression [ , ... ] }
 * [TABLESAMPLE](sql-ref-syntax-qry-select-sampling.html)
```

Comment:
Could you update the `FROM` syntax, too?
https://github.com/apache/spark/blob/b6229df16c02d9edcd53bc16ee12b199aaa0ee38/sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4#L574

Reply:
> Could you update the `FROM` syntax, too?
> https://github.com/apache/spark/blob/b6229df16c02d9edcd53bc16ee12b199aaa0ee38/sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4#L574

have updated:)